packages/cactus-test-tooling/src/main/typescript/corda/corda-test-ledger.ts

```diff
@@ -259,30 +259,105 @@ export class CordaTestLedger implements ITestLedger {
 
     const jars: ICordappJarFile[] = [];
 
-    {
-      const jarRelativePath = "contracts/build/libs/contracts-1.0.jar";
-      const jarPath = path.join(cordappRootDir, jarRelativePath);
-      this.log.debug(`Pulling jar file from container at: %o`, jarPath);
-      const jar = await Containers.pullBinaryFile(container, jarPath);
-      jars.push({
-        contentBase64: jar.toString("base64"),
-        filename: `${sampleCordapp}_contracts-1.0.jar`,
-        hasDbMigrations: false,
-      });
-      this.log.debug(`Pulled jar (%o bytes) %o`, jarPath, jar.length);
+    switch (sampleCordapp) {
```

Comment:
@elenaizaguirre Could the additional information needed on a per case basis get defined in another enum instead of writing a switch statement here? (I'm guessing the difference is the jar file name's version numbers so you could create an enum that maps instances of `SampleCordappEnum` to jar file names and then retain the much simpler code here wihch would also then benefit from the compiler at development time)

Reply:
@petermetz How about this? I have added this information to the one already available (sample root directory) in the enum in `sample-cordapp-enum.ts`. Do you prefer me to keep them separately or can we leave them in the same object?

Reply:
@elenaizaguirre Yeah that's perfect too, thank you very much for putting up with the nit-picks as usual.